src/soundRecorder.js

```diff
@@ -83,11 +83,13 @@ define(function (require) {
     this._inputChannels = 2;
     this._outputChannels = 2; // stereo output, even if input is mono
 
+    const workletBufferSize = 1024;
```

Comment:
I think we could determine the proper buffer size here, and abstract the code to do it into a shared method. Maybe something like

```
safeBufferSize(idealBufferSize) {
  if (<AudioWorklet is not supported>) return idealBufferSize
  else createScriptProcessorNode --> return its bufferSize
}
```

Reply:
@therewasaguy I've extracted this logic into a shared method in `helpers.js`: https://github.com/processing/p5.js-sound/pull/376/commits/2014fbc37baeda54075899cb4c3a32c6e42722fc#diff-b38f7254be1a9f817fd2e33c294481a1R305